apps/wekan/config.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "../schema.json",
+  "name": "Wekan",
+  "port": 8207,
+  "available": true,
+  "exposable": true,
+  "id": "wekan",
+  "tipi_version": 1,
+  "version": "7.26",
+  "categories": [
+    "development"
+  ],
+  "description": "Experience efficient task management with WeKan - the Open-Source, customizable, and privacy-focused kanban",
+  "short_desc": "Open-Source, customizable, and privacy-focused kanban",
+  "author": "Lauri Ojansivu",
+  "source": "https://github.com/wekan/wekan",
+  "form_fields": [
+    {
+      "type": "string",
+      "label": "email address",
+      "max": 50,
+      "min": 3,
+      "required": true,
+      "env_variable": "MAIL_ADDRESS"
+    },
+    {
+      "type": "password",
+      "label": "email password",
```

Comment:
Maybe use: `Email Password` to make it look better.

Reply:
done